airflow/auth/managers/simple/views/auth.py

```diff
@@ -78,17 +75,15 @@ def login_submit(self):
         next_url = request.args.get("next")
 
         found_users = [
-            user
-            for user in self.users
-            if user["username"] == username and self.passwords[user["username"]] == password
+            user for user in self.users if user[0] == username and self.passwords[user[0]] == password
```

Comment:
```suggestion
            user
            for user in self.users
            if user["username"] == username and self.passwords[user["username"]] == password
```

this probably needs to be switched back?